06 - Type Ahead/index-START.html

```diff
@@ -15,8 +15,68 @@
     </ul>
   </form>
 <script>
-const endpoint = 'https://gist.githubusercontent.com/Miserlou/c5cd8364bf9b2420bb29/raw/2bf258763cdddd704f8ffd3ea9a3e81d25e2c6f6/cities.json';
+  //// global variables
+  const endpoint = 'https://gist.githubusercontent.com/Miserlou/c5cd8364bf9b2420bb29/raw/2bf258763cdddd704f8ffd3ea9a3e81d25e2c6f6/cities.json';
+  let cities = [];
+  let searchTerm = '';
```

Comment:
does this variable need to be global?

Reply:
Nope! I moved it inside the scope of displayResults(). I just realised I do not use it anywhere else.